features/step_definitions/government_frontend_steps.rb

```diff
@@ -0,0 +1,36 @@
+Then /^I should see a radio button for "(.*?)"$/ do |value|
+  # We set visible to false since our radios are styled labels
+  page.body.should have_field("option", with: value, visible: false)
```

Comment:
For consistency, should this be `expect(page.body).to have_field("option", with: value, visible: false)`?